tests/e2e/java_test.go

```diff
@@ -32,7 +32,8 @@ var _ = Describe("odoJavaE2e", func() {
 		It("Should be able to deploy a git repo that contains a wildfly application", func() {
 
 			// Deploy the git repo / wildfly example
-			runCmd("odo create wildfly javaee-git-test --git " + warGitRepo)
+			cmpCreateLog := runCmd("odo create wildfly javaee-git-test --git " + warGitRepo + " -w")
+			Expect(cmpCreateLog).ShouldNot(ContainSubstring("This may take few moments to be ready"))
```

Comment:
I don't think it's a reliable way of detecting the status. Maybe try `oc get build <build-name>`?

Reply:
@mik-dass added validation using oc here https://github.com/redhat-developer/odo/pull/987/commits/e534ba62eac87b40728fe60310fe226388be232d